test/unit/message_test.rb

```diff
@@ -35,16 +41,16 @@ class MessageTest < ActiveSupport::TestCase
     assert_equal "team", appreciation.team
   end
 
-  test "index scope" do
+  test "visible scope" do
```

Comment:
[pronto-rubocop warning] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.